src/components/Tweet.js

```diff
@@ -1,34 +1,24 @@
-function Tweet() {
+import ProfileImage from "./ProfileImage";
+import User from "./User";
+import Timestamp from "./Timestamp";
+import Message from "./Message";
+import Actions from "./Actions";
+
+function Tweet(props) {
   return (
     <div className="tweet">
-      <img
-        src="https://i.imgur.com/9yw1Fyw.jpg"
-        className="profile"
-        alt="profile"
-      />
 
+      <ProfileImage image={props.tweet.user.image} />     
       <div className="body">
         <div className="top">
-          <span className="user">
-            <span className="name">Ironhack</span>
-            <span className="handle">@ironhack</span>
-          </span>
-
-          <span className="timestamp">Nov 30, 2020</span>
+        <User userData={props.tweet.user} />
+            <span className="handle">{props.tweet.user.handle}</span>
```

Comment:
You already moved <span> with user handle to User component, so do not need it here anymore